crates/goose/src/config/extensions.rs

```diff
@@ -55,6 +55,33 @@ impl ExtensionManager {
     pub fn set(name: &str, entry: ExtensionEntry) -> Result<()> {
```

Comment:
nit: i think we can instead remove the name arg from this method and read it off the entry